packages/git/src/node/git-backend-module.ts

```diff
@@ -74,6 +76,11 @@ export function bindRepositoryWatcher(bind: interfaces.Bind): void {
     bind(GitWatcherServer).toDynamicValue(context => context.container.get(DugiteGitWatcherServer));
 }
 
+export function bindPrompt(bind: interfaces.Bind): void {
+    bind(DugiteGitPromptServer).toSelf().inSingletonScope(); // TODO is this really singleton, probably not.
```

Comment:
it means that only the last page will be prompted, for single electron window should be alright.

Reply:
It is a leftover. It should not be a singleton. Thanks!